Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -25,7 +25,7 @@ public partial class GameTicker
         private TimeSpan _pauseTime;
 
         [ViewVariables]
-        public bool Paused { get; set; }
+        public new bool Paused { get; set; }
```

Comment:
wait what

Reply:
There is a Paused method on EntitySystem, so it is warning about it. By adding new, I am telling the compiler this is intentional.